runatlantis.io/docs/server-configuration.md

```diff
@@ -229,10 +229,13 @@ Values are chosen in this order:
   atlantis server --data-dir="path/to/data/dir"
   ```
   Directory where Atlantis will store its data. Will be created if it doesn't exist.
-  Defaults to `~/.atlantis`. Atlantis will store its database, checked out repos, Terraform plans and downloaded
+  Defaults to `/home/atlantis/`. Atlantis will store its database, checked out repos, Terraform plans and downloaded
```

Comment:
we should leave `~` since it means the user home folder and in each distribution this could be a different directory.
the `~` is pretty standard

Reply:
I think it's specific to the /home/ directory since the container image has a hard coded /home/atlantis

Reply:
Distros can each have their own 'HOME' variable specified in `/etc/default/useradd`, or `/home` by default according to the `useradd` command description that we run for the `atlantis` user in the docker-base.

I'm ok with moving it to `~/atlantis` if that makes things simpler, though it conflicts with the hardcoded /home/atlantis we chown/chmod.

Reply:
what I mean about the `~` is that no matter that the home dir is set in the distro it will always go to the home dir of the user.

Reply:
you know what, it is ok if we leave the `/home/atlantis`, it is easier to read.